### DISCUSSION TITLE
How to remove the weird names that appear on the top bar?

### DISCUSSION THREAD

**Discussion by txtyash** [zellij-org/zellij#1832] [Q&A] [answered]
I have these weird names that appear on the top bar of zellij
It looks like:
`Zellij (gruesome-position) > Tab #1 > Tab #2 >`
It takes up too much space for me. I would even like to remove the `Zellij` name and shorten the tab names to just the number. Basically a minimal tablline:
`1>2>3>4>`

**Comment by Lilja** [comment]
The "weird name" most probably be the auto generated session name. I don't know zellij too well but you might be able to reference your session with that name,

You can try `zellij -l compact` for a more compact layout.

**Reply by txtyash** [reply]
Already using `zellij -l compact` settings which doesn't fix my problem..

**Reply by Lilja** [reply]
After spending some time and reading the code, If you want to remove it, you must create your own plugin. Basically, you can apply a layout with zellij. A pane can have a custom plugin which loads the status bar the way you want it.

**Comment by pergamomo** [comment, marked as answer]
Is there an update to this?
It also takes up space in the Wezterm tabs.

It also looks like I lost the ability to rename Wezterm tabs completely when using Zellij.

**Reply by txtyash** [reply]
It seems to be that those names are session-names. Idm having default session names but the default naming for tabs should be `1/2/3...` instead of `tab #1/ tab #2/...` takes a lot of space...

**Reply by robrecord** [reply]
Workaround solution: name your sessions and your tabs in zellij. read the docs for how to do this. Eg. you could use just numbers for your sessions and tab names.

**Comment by Solarunit** [comment]
I would like to have an option to hide it as well.

**Comment by MarcinOrlowski** [comment]
You can hide session name by adding this to your config

```
 ui {
     pane_frames {
         hide_session_name true
     }
 }
```
